testing.json

```diff
@@ -0,0 +1,151 @@
+{ "type": "FeatureCollection",
+
+    "features": [
+      { 
+        "type": "Feature",
+        "geometry": {
+          "type": "Point", 
+          "coordinates": [-75.134729, 39.974192]
+          },
+          "properties": {
+            "name": "ArtWell",
+            "address": "100 W Oxford St",
+            "ZIP": "19122",
+            "website": "http://www.theartwell.org/",
+            "grant-amount": "$50,000"
```

Comment:
Consider keeping the data type for `grant-amount` as a raw number, ie, `50000` so that you can do numerical filtering or sorting on that data.  It can be reformatted for display purposes when shown in your web app.